styles.css

```
@@ -2,3 +2,5 @@
 @import url(components/header.css);
 @import url(components/section.css);
 @import url(components/footer.css);
+@import url(form.css);
+@import url(profil.css);
```

your file is called `profile.css` so this import currently doesn't work;

```suggestion
@import url(profile.css);
```